DEVNET.md

```diff
@@ -49,18 +49,39 @@ These steps have been used for [AWS](https://aws.amazon.com/) so maybe some chan
 
 ### VM instance
 
-Create a virtual machine to execute the component. The VM should have a running Docker daemon and should be able to pull an image from the chosen container registry.
+Create a virtual machine to execute the component. The VM should have a running Docker daemon and should be able to pull an image from the chosen container registry. These instructions asssume an Ubuntu image.
 
-[`How to install docker`](https://serverfault.com/questions/836198/how-to-install-docker-on-aws-ec2-instance-with-ami-ce-ee-update)
+#### Install AWS CLI
 
-#### tldr;
+Update the VM and install the AWS CLI:
 
 ```sh
-sudo yum update -y
-sudo yum install docker -y
-sudo service docker start
-sudo usermod -a -G docker <vm-user>
-sudo systemctl enable docker (restart docker when VM instance restarts)
+sudo apt update && sudo apt upgrade
+sudo snap install aws-cli --classic
+```
+
+#### Install Docker
+
+Install Docker:
+
+```sh
+curl -fsSL https://get.docker.com -o get-docker.sh
+sudo sh get-docker.sh
+```
+
+Add your user to the `docker` group if you want to run without `sudo`:
+
+```sh
+sudo usermod -aG docker ubuntu
+newgrp docker
+```
+
+The `ubuntu` user assumes the default user on Ubuntu images. The `newgrrp` command adds your user to the group without logging out and back in.
+
+Check that Docker is running:
+
+```sh
+docker run hello-world
 ```
 
 To grant access to the [ECR](https://aws.amazon.com/ecr/) registry from the VM execute `aws configure` with credentials that have `pull` permissions.
```

Comment:
Can you add a line referencing how the instance has to be added the role (and which role) to have ecr pull access?

Reply:
Yep! Added some details here: https://github.com/Lilypad-Tech/lilypad/blob/42504b47d8d7d78dcb1afda7ca6ecd62a4e419b8/DEVNET.md?plain=1#L54